js/client-ladder.js

```diff
@@ -86,6 +86,9 @@
 	});
 
 	this.LadderRoom = HTMLRoom.extend({
+		events: {
+			'keyup .searchinput': 'searchChange',
```

Comment:
Also, if you're looking for changes, `input` is probably the right event to listen to? It includes non-keyboard-based inputs, like right-click -> Paste.

Reply:
see above. theres a very non-zero change this was the wrong choice, i copied and pasted from teambuilder